From 7a31cc474f9d6d1b7826b99e5780195b82da76e0 Mon Sep 17 00:00:00 2001
From: Yang Shi <yang.shi@windriver.com>
Date: Fri, 14 Feb 2014 10:25:32 -0800
Subject: [PATCH 2/3] grsec: Fix build failure

Add missing ")" to fix the below build failure:

grsecurity/gracl_compat.c: In function 'copy_gr_arg_wrapper_compat':
grsecurity/gracl_compat.c:17:17: error: expected ')' before 'return'
                 return -EINVAL;
                 ^
grsecurity/gracl_compat.c:24:1: error: expected expression before '}' token
 }
 ^
grsecurity/gracl_compat.c:24:1: error: control reaches end of non-void function [-Werror=return-type]
 }
 ^

Signed-off-by: Yang Shi <yang.shi@windriver.com>
---
 grsecurity/gracl_compat.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/grsecurity/gracl_compat.c b/grsecurity/gracl_compat.c
index d79fa48..c2f2349 100644
--- a/grsecurity/gracl_compat.c
+++ b/grsecurity/gracl_compat.c
@@ -13,7 +13,7 @@ int copy_gr_arg_wrapper_compat(const char *buf, struct gr_arg_wrapper *uwrap)
                 return -EFAULT;
 
         if ((uwrapcompat.version != GRSECURITY_VERSION) ||
-	    (uwrapcompat.size != sizeof(struct gr_arg_compat))
+	    (uwrapcompat.size != sizeof(struct gr_arg_compat)))
                 return -EINVAL;
 
 	uwrap->arg = compat_ptr(uwrapcompat.arg);
-- 
1.8.5.2.233.g932f7e4

